### ISSUE TITLE
[Feature Request] add dynamic footer page transition icons like in the latest Tachiyomi app.

### ISSUE THREAD

**Issue by GoluLiliput** [LNReader/lnreader#62]

**Comment by rajarsheechatterjee** [comment]
Already exists.

![Screenshot_20210811-163632.jpg](https://user-images.githubusercontent.com/57822170/129021708-c4efde76-064d-4549-a5eb-e5d59ae1b882.jpg)

**Comment by GoluLiliput** [comment]
No...what I meant was animated icons at the bottom of the app like tachiyomi latest update

**Comment by Palloxin** [comment]
Could send a screenshot of what you are talking about? for clarity.
